lib/utilities/osal.c

```diff
@@ -12,6 +12,13 @@
 #include <errno.h>
 #include "pthread_queue.h"
 
+long os_get_tick(void) {
```

Comment:
I had added os_get_ms() and os_sleep_ms() last week because I didn't think "ticks" were as portable to Linux as milliseconds. It's not clear we need both. If we're going forward with os_get_tick() can you remove os_get_ms()?

Reply:
I think ms is more descriptive. Where did you add that? I didn't see it in master when I made these functions

Reply:
Looks like the implementation is on line 118 of osal.c
I added it with the initial S-band PR, because I thought I might need it for the pacer in sdr_driver.c